youtube/generic.md

```diff
@@ -34,13 +34,13 @@ Join Our **Community GitHub Organisation**:
 ## Social Media
 Follow on socials for behind the scenes footage and more.
 
-- Website: http://eddiejaoude.io 
+- Website: https://eddiejaoude.io 
```

Comment:
@krishdevdb oops, this is the link -
https://www.eddiehub.org as mentioned by @Vyvy-vi earlier

Reply:
I guess I forgot this 😅 Fixing It